core/templates/dev/head/domain/utilities/AutogeneratedAudioLanguageObjectFactory.ts

```diff
@@ -19,6 +19,14 @@
 import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
 
+export interface IAutogeneratedAudioLanguage {
```

Comment:
should we add the `Dict` keyword in the suffix to differentiate this with the domain object?

@bansalnitish, I'm not sure about the pattern we follow for the interface names, do you have any suggestion?

Reply:
There's no particular pattern that we are following but your suggestion seems reasonable to me.

Reply:
@DubeySandeep , thanks for the suggestion, I will definitely do so the next time around. And should I make the change you asked for?

Reply:
@sudiptapradhan1181, Yeah, let's fix this comment and then merge this PR. Thanks!

Reply:
Okay, will do so!

Reply:
@DubeySandeep , I made the changes you asked for. PTAL. The Travis test seems to be flaky. Can you please rerun it. Thanks!